karma_config/vueLocal.js

```diff
@@ -0,0 +1,35 @@
+const vue = require('../node_modules/vue');
```

Comment:
Could this be written as `const vue = require('vue');`? If not, could we add a comment explaining why?